hoodie-client/src/test/java/com/uber/hoodie/io/strategy/TestHoodieCompactionStrategy.java

```diff
@@ -108,10 +108,15 @@ public void testPartitionAwareCompactionSimple() {
     sizesMap.put(110 * MB, Lists.newArrayList());
     sizesMap.put(100 * MB, Lists.newArrayList(MB));
     sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+
     DayBasedCompactionStrategy strategy = new DayBasedCompactionStrategy();
     HoodieWriteConfig writeConfig = HoodieWriteConfig.newBuilder().withPath("/tmp").withCompactionConfig(
-        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy).withTargetIOPerCompactionInMB(400).build())
-        .build();
+        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy)
+            .withTargetPartitionsPerDayBasedCompaction(2)
+            .build()).build();
     List<CompactionOperation> operations = createCompactionOperations(writeConfig, sizesMap);
```

Comment:
Can you add an assertion that only compaction operations for last 2 days are selected ?